File: HA2_1.ts
import * as readline from 'readline';

function Display(iNo : number)
{
    let iCnt : number = 0

    for(iCnt=0; iCnt < iNo; iCnt++)
    {
        console.log("*")
    }

}

async function main()
{
    const rl = readline.createInterface({
        input:process.stdin,
        output: process.stdout
    })

    let iValue : number = 0;

    iValue = parseInt(await askQuestion(rl, "Enter a Number :")) ||0;

    Display(iValue)

    rl.close()
}

function askQuestion(rl: readline.Interface, question: string): Promise<string>
{
    return new Promise((resolve) =>
    {
        rl.question(question,(answer : any) =>{
            resolve(answer)
        })
    })
}
main()

File: HA2_1.java
//Accept one number from user and print that number of * on screen
import java.util.*;

class Number
{
    public void Display(int iNo)
    {
        int iCnt = 0;
        for(iCnt =1; iCnt<=iNo; iCnt++)
        {
            System.out.println("*");
        }
    }
}
public class HA2_1
{
    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter a number");
        int iValue = sobj.nextInt();

        Number nobj = new Number();
        {
            nobj.Display(iValue);
        }
        sobj.close();
    }
}